content/sensu-go/5.14/getting-started/learn-sensu.md

```diff
@@ -82,22 +81,21 @@ admin      cluster-admins   true
 agent      system:agents    true    
 {{< /highlight >}}
 
-Sensu keeps track of monitored components as entities.
-Let's start by using sensuctl to make sure Sensu hasn't connected to any entities yet:
+Sensu keeps track of monitored components as entities. Start by using sensuctl to make sure Sensu hasn't connected to any entities yet:
 
 {{< highlight shell >}}
 sensuctl entity list
  ID   Class   OS   Subscriptions   Last Seen  
 ──── ─────── ──── ─────────────── ─────────── 
```

Comment:
The second time I ran through the sandbox (and subsequent times), the entity list includes sensu-go-sandbox (in other words, the initial `sensuctl entity list` isn't empty). I think we should add a cleanup/delete command at the end of this sandbox lesson.

Reply:
Most likely stuff that remains in `etcd`. As the lesson is meant to be done from start to finish, I think having them blow away the VM is fine. Secondary runs will work quicker as they won't have to re-download the base image of the OS.

So `vagrant destroy`.

Reply:
Thank you! I kept trying to do it with sensuctl and sensuctl was like "idk what you want 🤷‍♀ "